### ISSUE TITLE
#2833: Projects: Table (Tab) Ordering for Projects

### ISSUE THREAD

**Issue by tcarolan** [datagrok-ai/public#2833]
Table (Tab) ordering was partially addressed by maintaining consistent loading of the tables in the order they were added to the workspace.  I've encountered problems were this is not maintained and may be related to other issues with Data Query loaded tables and table names not being maintained (see other recent bug reports). 

There needs to be a way for tabs to be reordered and that ordering saved to a Project.  And when the project is reloaded, the order is maintained.  Users will typically try to drag the tab to the left or right, but then the visualization pops out and the user has no idea how to recover (potentially a lot of work lost).

Maybe there could be a dialog similar to the "tables" and "columns" dialog called "tabs" that would allow the user to reorder these tabs before saving a project. This is one suggestion.

**Comment by dnillovna** [comment]
This issue has been mirrored in Jira: https://reddata.atlassian.net/browse/GROK-15599

**Comment by skalkin** [comment]
Good point, we discussed it in details with @LaykDimon recently, and he is working on it

**Comment by LaykDimon** [comment]
also take the current order when we try to save the project
